woocommerce/woocommerce
# Made two strings translatable in slotfill.js

Made two strings ('Coming soon' and 'Live') translatable in slotfill.js

### How to test the changes in this Pull Request:

1. Create a new site
2. Go to Site Settings > Language and set the site language to a language other than English such as `Français` (Some languages may not have the strings translated)
3. Go to /wp-admin/update-core.php and update the translations (Click on the button under the `Traductions` section)
4. Go to /wp-admin/admin.php?page=wc-settings&tab=site-visibility
5. Confirm it is translated to the selected language (except for `Coming soon` which is not translated yet)

![Screenshot 2024-08-06 at 15 48 39](https://github.com/user-attachments/assets/5f821ea8-5acd-4c67-a428-1595945030f9)

### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 
Made two strings translatable in slotfill.js
</details>
